### ISSUE TITLE
I want to use Google.OrTools for .NET 7.0, how can I build it?

### ISSUE THREAD

**Issue by hexianggui** [google/or-tools#3866]
ll of my projects are in .NET 7.0, but I have a NuGet package that is designed for 6.0. Occasionally, I encounter an error: 'Could not load file or assembly 'Google.OrTools, Version=9.6.2534.0, Culture=neutral, PublicKeyToken=37e4c4e778bf1216'

**Comment by CADBIMDeveloper** [comment]
Just for the information, I have no problems to build an application for .NET 7 which is using the latest ortools (9.6) it works for me both on Windows (my dev machine) and in the cloud inside a docker container on Linux machine

**Comment by hexianggui** [comment]
Yes, initially, I had no issues when running the application. However, recently, after the first few calls or so, I started encountering an error: "Could not load file or assembly 'Google.OrTools, Version=9.6.2534.0, Culture=neutral, PublicKeyToken=37e4c4e778bf1216'." I'm not sure of the exact reason behind this
